packages/runner/src/runner.ts

```diff
@@ -90,10 +93,21 @@ export const run = (name: string = process.argv[2]): void | Promise<void> => {
 			cwd = path;
 		},
 		execute(command: string, args: string[] = [], env?: object): Promise<CommandResult> {
-			return execute(command, args, {
+			const prom = execute(command, args, {
 				cwd,
 				env: env as NodeJS.ProcessEnv,
 			}, log);
+			prom.then((result: CommandResult) => {
```

Comment:
Should return this instead of prom below

Reply:
Interesting. It seems to work both ways. Is it a convenience thing?

Reply:
<img width="714" alt="screen shot 2019-03-06 at 6 12 36 pm" src="https://user-images.githubusercontent.com/10180857/53920501-6c8def00-403b-11e9-8873-0d7f748f09a2.png">

Can't.

Reply:
You can add
```
return result;
```
to the end of that

Reply:
Not sure what you mean. To the end of what?

Reply:
Inside the `prom.then` block

Reply:
ah very fancy.